applications/SolidMechanicsApplication/python_scripts/process_handler.py

```diff
@@ -158,11 +159,9 @@ def LoadsList(self):
     #
     def ConstructProcess(self, process):
 
-        kratos_module = __import__(process["kratos_module"].GetString())
-        python_module = __import__(process["python_module"].GetString())
+        python_module = import_module(process["kratos_module"].GetString() + "." + process["python_module"].GetString())
```

Comment:
This should use the process factory from the core... I cannot approve this anyway, I am not involved in the Solid app